### PR TITLE
Update translation to v2.7.0

### PR DESCRIPTION
**Pull request by chenghuzi**
As titled.

Notice that main changes are two recently added tutorial translations. Some redundant comments are removed. 

Autogenerated parts are not included in this update though some of the autogenerated `.po` files are updated as the timestamps are changed.

### CODE REVIEW

File: source/locale/zh_CN/LC_MESSAGES/tutorial/20_recipes/002_multi_objective.po [hunk 1]
```diff
@@ -29,10 +29,11 @@ msgid "Multi-objective Optimization with Optuna"
...
 
 #: ../../source/tutorial/20_recipes/002_multi_objective.rst:17
+#, fuzzy
 msgid ""
 "This tutorial showcases Optuna's multi-objective optimization feature by "
-"optimizing the validation accuracy of MNIST dataset and the FLOPS of the "
-"model implemented in PyTorch."
+"optimizing the validation accuracy of Fashion MNIST dataset and the FLOPS"
+" of the model implemented in PyTorch."
 msgstr "本教程借由同时优化 PyTorch 编写的模型在 MNIST 数据集上的验证集准确度以及 FLOPS 来展示 Optuna 的多目标优化特性."
```
Comment (belldandyxtq) [inline source/locale/zh_CN/LC_MESSAGES/tutorial/20_recipes/002_multi_objective.po]: Fasion MNIST

File: source/locale/zh_CN/LC_MESSAGES/tutorial/20_recipes/009_ask_and_tell.po [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+msgid "Ask-and-Tell Interface"
+msgstr "Ask-and-Tell 接口"
+
+#: ../../source/tutorial/20_recipes/009_ask_and_tell.rst:17
+msgid ""
+"Optuna has an `Ask-and-Tell` interface, which provides a more flexible "
+"interface for hyperparameter optimization. This tutorial explains three "
+"use-cases when the ask-and-tell interface is beneficial:"
+msgstr "Optuna 带有 `Ask-and-Tell` 接口, 它为超参数优化提供了一个更灵活的接口."
+"本教程展示了, 在三种情况下 ask-and-tell 接口是有益的."
```
Comment (belldandyxtq) [inline source/locale/zh_CN/LC_MESSAGES/tutorial/20_recipes/009_ask_and_tell.po]: 本教程将展示三种可以用到ask-and-tell接口的情况

File: source/locale/zh_CN/LC_MESSAGES/tutorial/20_recipes/009_ask_and_tell.po [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+msgid ""
+"This interface is not flexible enough. For example, if ``objective`` "
+"requires additional arguments other than ``trial``, you need to define a "
+"class as in `How to define objective functions that have own arguments? "
+"<../../faq.html#how-to-define-objective-functions-that-have-own-"
+"arguments>`_. The ask-and-tell interface provides a more flexible syntax "
+"to optimize hyperparameters. The following example is equivalent to the "
+"previous code block."
+msgstr "这个接口并不灵活. 比如, 如果 ``objective`` "
+"需要不同于 ``trial`` 的额外参数, 你就需要顶一个类, 如同 `How to define objective functions that have own arguments? "
```
Comment (belldandyxtq) [inline source/locale/zh_CN/LC_MESSAGES/tutorial/20_recipes/009_ask_and_tell.po]: 你就需要定义一个类，例如

File: source/locale/zh_CN/LC_MESSAGES/tutorial/20_recipes/009_ask_and_tell.po [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+#: ../../source/tutorial/20_recipes/009_ask_and_tell.rst:276
+msgid "Batch Optimization"
+msgstr "批优化"
+
+#: ../../source/tutorial/20_recipes/009_ask_and_tell.rst:277
+msgid ""
+"The ask-and-tell interface enables us to optimize a batched objective for"
+" faster optimization. For example, parallelizable evaluation, operation "
+"over vectors, etc."
+msgstr "为了更快的优化, ask-and-tell 接口使我们可以优化一个批目标函数. 比如, 并行求值, 向量操作等."
```
Comment (belldandyxtq) [inline source/locale/zh_CN/LC_MESSAGES/tutorial/20_recipes/009_ask_and_tell.po]: 为了更快地完成优化, ask-and-tell 接口支持优化批目标函数.

File: source/locale/zh_CN/LC_MESSAGES/tutorial/20_recipes/010_reuse_best_trial.po [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+
+#: ../../source/tutorial/20_recipes/010_reuse_best_trial.rst:15
+msgid "Re-use the best values"
+msgstr "重新使用最佳值"
+
+#: ../../source/tutorial/20_recipes/010_reuse_best_trial.rst:17
+msgid ""
+"In some cases, you may want to re-evaluate the objective function with "
+"the best hyperparameters again after the hyperparameter optimization."
+msgstr "有时候, 在完成超参数优化以后, 你可能要用最佳的超参数对对目标函数重新求值."
```
Comment (belldandyxtq) [inline source/locale/zh_CN/LC_MESSAGES/tutorial/20_recipes/010_reuse_best_trial.po]: 有时候, 在完成超参数优化以后, 你可能要用最佳的超参数对~对~目标函数重新求值.

**Review by belldandyxtq — CHANGES_REQUESTED** [review]

**Comment by belldandyxtq** [comment]
Next time, can you please separate `files with translation update` and `files without translation update` into different PRs? So that we do not have a huge diff to look into and try to find the update. For those `files without translation update` we can just merge.

**Comment by belldandyxtq** [comment]
And for those `needs translation update but will do in another PR`. Can you please include them when the translation is ready so that we do not forget to translate

File: source/locale/zh_CN/LC_MESSAGES/tutorial/20_recipes/009_ask_and_tell.po (hunk 3 above)
Comment (chenghuzi) [reply source/locale/zh_CN/LC_MESSAGES/tutorial/20_recipes/009_ask_and_tell.po]: I think "就像...里做的那样" might be better

Comment (belldandyxtq) [reply source/locale/zh_CN/LC_MESSAGES/tutorial/20_recipes/009_ask_and_tell.po]: I was trying to point out a typo, "顶一个类"; I think your change is also OK

**Review by belldandyxtq — APPROVED** [review]

**Pull request merged**